controller/internal/supervisor/iptablesctrl/acls.go

```diff
@@ -287,6 +293,27 @@ func (i *Instance) generateACLRules(contextID string, rule *aclIPset, chain stri
 	return iptRules, reverseRules
 }
 
+// programExtensionsRules programs iptable rules for the given instruction
+func (i *Instance) programExtensionsRules(rule *aclIPset, chain, proto string) error {
+
+	if byteCodes, ok := rule.extensions["bpf"]; ok {
+		for _, byteCode := range byteCodes {
```

Comment:
I think we can make this more generic. Don't need to just be the bpf extension. It is essentially a list of parameters that can be blidly passed to iptables without us looking at them. Opaque at this point.Essentially the caller will provide a string array that will include "-m" "bpf" ... or whatever else .. we don't need to hardwire these values.